res/database/Default/user/cujo.xml

```diff
@@ -21,7 +21,7 @@
 				<!-- "morgen oder uebermorgen, 12-16 Uhr" -->
 				<effect trigger="happen" type="triggernews" time="2,1,2,12,16" news="cujo-news-superkerl-02" />
 			</effects>
-			<data genre="1" price="1.0" quality="10" flags="2"/>
+			<data genre="1" price="1.0" quality="10" />
```

Comment:
That news arleady appears quite often. Probably because the follow-up entries are not unique (editor warning). We still have this "problem" with many news items, we simply don't have enough news content to go around. Variables for this are on my todo (Wonderwoman...).

Reply:
If it appears multiple times, it should be a bug - unique events should appear only once.
I'd say, remove the flag together with introducing variables.